tidb-lightning/tidb-lightning-backends.md

```diff
@@ -0,0 +1,471 @@
+---
+title: TiDB Lightning Backends
+summary: Learn the backends of TiDB Lightning.
+aliases: ['/docs/dev/tidb-lightning/tidb-lightning-tidb-backend/','/docs/dev/reference/tools/tidb-lightning/tidb-backend/','/tidb/dev/tidb-lightning-tidb-backend']
+---
+
+# TiDB Lightning Backends
+
+TiDB Lightning supports the following [backends](/tidb-lightning/tidb-lightning-glossary.md#back-end):
+
++ [Importer-backend](#tidb-lightning-importer-backend) (default)
++ [Local-backend](#tidb-lightning-local-backend)
++ [TiDB-backend](#tidb-lightning-tidb-backend)
+
+The backend determines how `tidb-lightning` imports data into the target cluster.
```

Comment:
To be user-friendly: State what "backend" can do first, and say what backends Lightning supports.

Reply:
Done